packages/next/link.d.ts

```diff
@@ -1,2 +1,2 @@
 export * from './dist/client/link'
-export { default } from './dist/client/link'
+export { default, LinkProps } from './dist/client/link'
```

Comment:
This is already exported in the line above and shouldn't be required.

Reply:
Correct, thanx